core/models.py

```diff
@@ -225,5 +256,8 @@ def post_save_score_job(sender, instance, *args, **kwargs):
 class ScoreResult(models.Model):
     score_job = models.ForeignKey(ScoreJob, on_delete=models.CASCADE, blank=True, null=True)
     created = models.DateTimeField(default=timezone.now)
+    #  setting the upload directory and file name
     data = models.FileField(upload_to='scores')
     log = models.FileField(upload_to='scores_logs', null=True, blank=True)
+    overall_score = models.FloatField(null=True, blank=True, validators=[MaxValueValidator(1.0), MinValueValidator(0.0)])
+    result_type = models.CharField(max_length=10, null=True, blank=True)
```

Comment:
We want to have the possible values for `result_type` defined in the model.  For instance:
```
    class ResultTypes(models.TextChoices):
        SIMPLE = 'simple', _('Direct value')
        ROC = 'roc', _('Receiver Operating Characteristic')
```
and then
```
result_type = models.CharField(max_length=10, choices=Status.choices, null=True, blank=True)
```